flink-end-to-end-tests/flink-tpcds-test/tpcds-tool/README.md

```diff
@@ -0,0 +1,28 @@
+# TPC-DS tool
+
+This directory contains the necessary scripts/files to run a TPC-DS test.
+
+## data_generator.sh scripts
+
+Scripts that to download the TPC-DS data generator and generate test data, the supported OS including GNU/Linux OS and Mac OS.
+
+* parameter 1: generator_dir, the directory to save TPC-DS data generator.
+* parameter 2: scale_factor, the TPC-DS test scale factor, we use the validation scale factor 1 (1G) by default.
```

Comment:
Scale factor indicates raw data size in GB,  we use the validation scale factor 1 (1G) by default.